docs/pages/repo/docs/reference/configuration.mdx

```diff
@@ -292,3 +295,23 @@ Set type of output logging.
   }
 }
 ```
+
+### `persistent`
+
+`type: boolean`
+
+Label a task as `persistent` if it is a long-running process, such as a dev server or `--watch` mode.
```

Comment:
I'd add another sentence here describing why this option is necessary.

Reply:
done